src/tests/Microsoft.PowerFx.Interpreter.Tests/Scenarios/DataTableMarshallerProvider.cs

```diff
@@ -6,6 +6,8 @@
 using System.Collections.Generic;
 using System.Data;
 using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
nit - do we need this?

Reply:
No. It probably ended up there while I was paying with some old version code. Thanks for catching this.